packages/commons-ui-core/src/NavBar/index.js

```diff
@@ -0,0 +1,3 @@
+import NavBar from "./NavBar";
+
+export default NavBar;
```

Comment:
So far we have `NavBar` exported twice. I know this is the approach you recommended, just curious if the double exportation is also intended

Reply:
I only see a single `export` @gertie-sheshe but I kinda see what you mean. eslint doesn't allow use to export default [directly](https://eslint.org/docs/rules/no-restricted-exports#default-exports) similar to how [MUI](https://github.com/mui/material-ui/blob/dccbdbb25e0e335b8c1e0db13ec17ff0bd1ae7af/packages/mui-material/src/AppBar/index.js#L1) does it.